hosting/src/app/widgets/chat/chat.component.scss

```diff
@@ -43,6 +43,7 @@
                 font-size: 1em;
                 appearance: none;
                 padding: $input-padding ($input-padding * 1.5);
+                padding-right: 55px;
```

Comment:
should be using `em` for this as its due to text that will scale on the page (not fixed size)

also it could (probably should) be combined into the single line padding on the row above

the base for em is 14px.... so 55/14 = approx 4 and input-padding is 0.375 so 4/0.375 = approx 10 as such this probably works.... but might need a visual check to be sure

```
padding: $input-padding ($input-padding * 10) $input-padding ($input-padding * 1.5)
```

Reply:
Done. That values work  as expected in my tests.

Have you considered letting the chat input grow to a multiline area if the message is too long to fit in the current one? letting the user see the whole time the full message. That's another thing we could add

Reply:
interesting idea, should you change the Dom between type"input" and type"textarea" or just start it as "textarea" but have a special attribute on there like `"grow"=true` so it starts single line high, also can this be done just with CSS (never looked into a feature like this before) or would javascript need to know its overflowing... obviously we want to disable the user resizing this input `resize: none` I would assume.... one for the backlog I think